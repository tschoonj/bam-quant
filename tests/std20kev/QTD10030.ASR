File: C:\xrfdaten\2014KW30\chris\std20kev\std10030.spe
Start: 21.07.2014 18:14:45
Stop: 21.07.2014 18:16:26
BESSY: 298.5954104 298.2115183
IONI: 3.230873E-11 3.155663E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 72394
Run: 100000
Dead: 28
ICR: 95540
OCR: 0
Accumulated BESSY: 59668.139668
Accumulated Ioni: 6.38092925E-09
N Accumulation: 200
Normfactor:  4.3295573e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zn
exp_x 131.260406494141 131.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  30   1      8.631   5975333.      2445.     176.61
