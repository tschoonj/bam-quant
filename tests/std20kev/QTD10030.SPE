$DATA:
0 2000
16864
11146
4872
1544
346
71
19
10
7
4
3
3
0
1
2
10
156
1311
1780
1716
1546
1594
1469
1403
1424
1332
1290
1231
1213
1136
1191
1150
1128
1106
1023
1013
981
991
947
938
926
929
884
966
913
823
918
844
838
842
854
766
776
785
783
752
768
742
768
800
748
788
798
772
714
743
706
747
710
719
710
717
699
685
678
678
707
640
647
692
644
627
637
636
616
655
658
585
641
606
622
646
639
576
644
642
615
621
658
665
599
672
667
669
634
688
651
603
638
591
621
578
604
589
554
591
588
541
554
578
554
565
604
590
575
548
546
536
547
606
598
632
536
564
539
529
573
548
544
543
509
602
551
559
630
633
645
655
657
624
632
601
534
571
550
524
561
556
571
548
526
533
524
507
560
499
489
490
489
550
525
522
525
513
521
520
534
520
505
498
487
544
498
483
522
519
467
513
527
516
487
485
498
468
526
495
511
556
505
508
515
511
514
495
512
482
471
511
511
481
504
527
506
488
497
529
525
496
519
498
516
439
481
461
489
490
520
466
500
484
505
512
549
514
481
513
503
485
514
508
465
549
484
500
523
528
495
528
499
491
564
512
501
497
499
503
539
512
510
509
529
522
493
490
520
569
541
565
555
527
561
534
573
515
556
538
526
584
538
534
555
550
557
549
574
570
546
563
535
581
559
530
586
567
587
591
588
605
587
572
574
579
599
610
631
625
606
599
659
603
631
685
743
673
706
778
810
816
792
836
732
690
707
713
707
695
690
677
737
726
724
753
800
819
883
1033
1174
1378
1583
1805
1890
1986
1845
1747
1537
1289
1159
977
953
911
890
836
871
899
899
885
901
930
946
908
930
951
970
973
1029
1031
1025
988
1005
1016
1029
1049
1087
1050
1003
969
1023
1112
1038
1033
1118
1147
1183
1236
1225
1277
1300
1307
1371
1396
1470
1413
1401
1415
1519
1506
1576
1707
1661
1792
1857
2001
2127
2138
2342
2374
2518
2683
2858
3076
3192
3505
3980
4429
5204
6687
9135
14704
25214
45708
80835
137408
216483
319976
436760
548966
635616
679685
667219
603390
500328
383295
270058
176120
105782
59172
31039
15707
7856
4312
2872
2036
1726
1574
1480
1476
1413
1295
1304
1316
1315
1396
1500
1555
1657
1733
1850
1976
2182
2310
2660
2988
3532
4367
5853
8671
13048
20312
30622
44385
60054
77338
90140
100959
102746
97733
85834
69600
52588
37076
24151
14672
8287
4482
2303
1148
577
305
173
102
88
84
66
60
60
67
57
45
43
37
44
46
30
47
50
33
48
32
34
36
37
36
26
26
23
32
27
30
25
25
27
21
40
34
29
28
22
25
27
20
29
25
22
13
22
17
19
25
13
22
23
13
23
15
17
16
21
14
13
26
16
8
14
13
17
23
15
15
15
16
18
15
13
12
14
15
15
20
13
10
14
20
16
17
10
15
11
22
21
17
13
10
13
14
19
22
9
17
16
12
12
10
10
7
12
12
10
12
13
13
14
10
10
15
12
13
15
2
17
13
15
10
6
13
17
15
10
14
8
10
11
8
11
22
15
13
18
14
13
8
10
13
14
5
13
16
11
11
19
11
9
11
10
6
10
9
17
7
11
7
14
13
7
9
12
9
8
9
13
11
15
7
13
8
11
7
3
13
7
14
15
8
7
11
10
12
8
9
10
9
7
11
6
9
7
11
12
10
15
8
5
9
14
8
16
9
7
19
10
8
9
12
10
9
13
9
8
6
14
8
14
15
5
10
13
6
7
7
11
10
11
12
10
17
8
11
10
9
9
8
18
8
13
13
13
10
10
12
9
8
13
8
12
16
10
17
19
9
6
13
13
8
8
14
14
13
13
7
16
13
16
31
27
19
20
24
31
28
26
30
33
33
41
31
35
33
43
41
43
32
42
41
46
35
33
31
36
30
27
21
26
15
20
15
16
12
13
19
14
21
20
25
21
15
12
22
14
20
18
19
26
22
22
32
13
25
16
23
26
33
27
24
22
22
24
39
20
32
32
36
23
28
36
45
42
39
62
44
76
85
118
163
242
367
539
758
1042
1451
1784
2275
2883
3191
3408
3738
3782
3636
3295
3009
2593
2123
1680
1229
944
688
394
266
174
113
90
54
28
36
31
36
43
34
35
20
37
30
37
42
37
42
51
64
67
85
109
141
199
265
365
434
553
659
818
923
1063
1088
1169
1127
1081
956
856
639
606
472
370
205
183
116
86
48
33
31
11
14
13
10
8
10
16
15
11
12
11
15
12
17
13
20
25
14
33
25
29
33
36
57
36
64
73
70
88
103
116
92
91
90
89
80
83
59
56
54
35
35
38
37
27
36
27
29
27
35
29
35
30
32
37
44
36
41
42
53
62
48
55
54
62
62
78
84
82
84
90
94
89
103
98
85
91
74
78
56
74
62
58
50
53
44
28
21
21
12
8
12
12
8
2
5
2
3
1
0
1
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
1
0
0
1
0
0
0
1
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
1
0
0
0
0
1
1
1
0
1
0
0
0
0
0
0
0
0
0
2
1
0
0
1
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
1
1
2
0
0
0
2
0
0
1
0
0
0
0
0
0
1
0
1
0
1
0
0
1
0
1
2
3
0
1
2
2
6
4
7
3
12
15
17
23
18
9
12
15
16
13
9
18
6
7
11
5
6
0
1
2
1
1
0
0
0
1
0
0
0
0
0
0
1
1
2
0
3
0
1
0
1
2
0
3
5
4
1
5
5
5
5
6
6
2
5
5
2
4
8
4
1
1
1
2
1
1
2
0
0
1
0
0
1
0
0
1
1
0
0
1
0
0
1
0
1
1
0
1
1
0
1
0
2
0
0
3
0
0
2
2
0
2
1
1
2
1
0
0
1
1
0
0
1
0
1
0
1
0
0
0
0
0
2
1
0
2
1
1
0
0
1
0
0
0
0
2
0
1
2
3
1
3
2
1
1
1
0
1
2
0
1
1
0
1
0
0
1
0
0
0
0
0
0
0
0
0
2
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
1
2
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10030.spe
Start: 21.07.2014 18:14:45
Stop: 21.07.2014 18:16:26
BESSY: 298.5954104 298.2115183
IONI: 3.230873E-11 3.155663E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 72394
Run: 100000
Dead: 28
ICR: 95540
OCR: 0
Accumulated BESSY: 59668.139668
Accumulated Ioni: 6.38092925E-09
N Accumulation: 200
Normfactor:  4.3295573e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zn
 
exp_x 131.260406494141 131.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  30   1      8.631   5975333.      2445.     176.61
