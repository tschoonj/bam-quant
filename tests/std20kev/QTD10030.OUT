     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:44  บ   
     บ  Spectrum: QTD10030.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =   156.4   last change =   .00%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -39.3 ๑   100.0   บ     -39.3 ๑      .4   บ   
     บ  GAIN (eV/ch)  บ     20.247 ๑   2.000   บ    20.247 ๑    .001   บ   
     บ det NOISE (eV) บ      -86.4 ๑    40.0   บ     -86.4 ๑    -2.4   บ   
     บ  FANO factor   บ       .193 ๑    .050   บ      .193 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Zn-Ka                           5975333. ๑   2445.           บ   
     บ    KA1         8.639    .66048     3946565. ๑   1615.    176.6  บ   
     บ              428.627    167.92         305.           9.65E-01  บ   
     บ    KA2         8.616    .33952     2028768. ๑    830.    173.5  บ   
     บ              427.491    167.64         306.           9.65E-01  บ   
     บ                                                                 บ   
     บ  2 Zn-Kb                            956041. ๑    978.           บ   
     บ    KB1         9.572   1.00000      956041. ๑    978.     57.6  บ   
     บ              474.709    179.02         266.           9.74E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           61866. ๑    249.           บ   
     บ    L1N2       17.262    .74313       45975. ๑    186.     19.1  บ   
     บ              854.541    252.57          80.           9.84E-01  บ   
     บ    L1O3       18.203    .23784       14714. ๑     60.      3.5  บ   
     บ              901.008    260.15          61.           9.77E-01  บ   
     บ    L1O2       19.144    .01903        1177. ๑      8.      5.4  บ   
     บ              947.475    267.51          49.           9.66E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
